### PR TITLE
[go-gen] Enquote

### PR DESCRIPTION
**Pull request by c-harding**
Add function to wrap in quotes

### CODE REVIEW

File: src/main/scala/temple/generate/utils/CodeTerm.scala [hunk 1]
```diff
@@ -106,12 +106,16 @@ object CodeTerm {
       object block extends codeWrap("{\n", "}\n")
     }
 
-    object square extends codeWrap("[", "]") {
+    object square extends codeWrap("[" ush, "]") {
```
Comment (willjvsmith) [inline src/main/scala/temple/generate/utils/CodeTerm.scala]: what dis

**Comment by willjvsmith** [comment]
I think a better solution would be a generalised StringUtils function, that defaults to double quotes, but allows you to supply the string to wrap with, rather than only allowing JSON-encoding a string. I don't think `CodeTerm.scala` is the right place for it.

**Comment by willjvsmith** [comment]
I'll take over this PR tomorrow when we're inside working hours 😉

File: src/main/scala/temple/generate/utils/CodeTerm.scala [hunk 2]
```diff
@@ -109,9 +109,15 @@ object CodeTerm {
     object square extends codeWrap("[", "]") {
       object block extends codeWrap("[\n", "]\n")
     }
-
-    object singleQuotes extends codeWrap("'", "'")
-    object doubleQuotes extends codeWrap("\"", "\"")
-    object españolQue   extends codeWrap("¿", "?")
```
Comment (Lewiky) [inline src/main/scala/temple/generate/utils/CodeTerm.scala]: ¿Por qué no hay amor para los españoles?

**Review by Lewiky — APPROVED** [review]

File: src/main/scala/temple/generate/utils/CodeTerm.scala (hunk 2 above)
Comment (c-harding) [reply src/main/scala/temple/generate/utils/CodeTerm.scala]: Will está a cargo ahora, puede agregarlo de nuevo si quiere

Comment (willjvsmith) [reply src/main/scala/temple/generate/utils/CodeTerm.scala]: español está de vuelta en el menú

**Pull request merged**